internal/lockclient/client.go

```diff
@@ -4,7 +4,18 @@ import "github.com/SystemBuilders/LocKey/internal/lockservice"
 
 // Client describes a client that can be used to interact with
 // the Lockey lockservice. The client can start the lockservice
-// and interact acquire and release locks by making calls to it.
+// and interact by making calls to it.
+//
+// The client has the ability to start the lockservice from its
+// in-built function or it can be started separately.
+//
+// The client offers the user to Acquire a lock, Release a lock,
+// and Watch or Pounce on any object using it's descriptor.
```

Comment:
watch and pounce aren't there rn so maybe we should remove this from this comment for now?

in this context 'its' should be used not "it's"

Reply:
Done